src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -576,4 +576,10 @@
   <data name="Predefined_conversion_from_0_to_1" xml:space="preserve">
     <value>Predefined conversion from {0} to {1}.</value>
   </data>
+  <data name="Fixallscope_containingtype_and_fixallscope_containingmember_are_not_supported_with_this_constructor" xml:space="preserve">
+    <value>'FixAllScope.ContainingType' and 'FixAllScope.ContainingMember' are not supported with this constructor</value>
```

Comment:
so the loc strings should match the actual english content as much as possible.  so that means matching cases as well.